### PR TITLE
add Eyedropper

### PR DESCRIPTION
**Pull request by FineFindus**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.finefindus.eyedropper.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "eyedropper",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--env=RUST_LOG=eyedropper=debug",
```
Comment (hfiguiere) [inline com.github.finefindus.eyedropper.json:12]: debug logging?

**Comment by hfiguiere** [comment]
bot, build com.github.finefindus.eyedropper

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.github.finefindus.eyedropper.json (hunk 1 above)
Comment (FineFindus) [reply com.github.finefindus.eyedropper.json]: oops, I forgot to remove it. It's removed now.

**Pull request closed without merging**